app/views/guests/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h3><%= @guest.name %></h3>
+<p>rooms:</p>
+<ul>
+  <% @guest.rooms.each do |room| %>
+  <p></p>
+  <li><%= room.suite %></li>
+  <li><%= room.rate %>$ per night</li>
+  <li><%= room.hotel.name %> hotel</li>
+  <% end %>
+</ul>
+
+<%= form_with url: "/guests/#{@guest.id}", method: :patch, id: 'Add Room' do |f| %>
+  <h3>Select room:</h3>
+  <%= f.label :room_id %>
+  <%= f.text_area :room_id%>
+  <p>Submit this room: <%= f.submit "Submit"%></p>
+  </form>
+<% end %>
```

Comment:
This form looks good, however, since in your routes, you have `PATCH /guests/:id` going to the `guest_rooms#create` action, this might make more sense to have this go to a `POST` since you're creating a new `GuestRoom` in the database. It could look something like `POST /guest_rooms, to: 'guest_rooms#create'`. And then, you'll need to pass in your `guest_id` and `room_id` via your form (hidden fields might help you pass your guest id through).